x/supplier/client/cli/tx_stake_supplier.go

```diff
@@ -54,3 +67,32 @@ $ pocketd --home=$(POCKETD_HOME) tx supplier stake-supplier 1000upokt --keyring-
 
 	return cmd
 }
+
+// TODO_BLOCKER, TODO_HACK: The supplier stake command should take an argument
+// or flag that points to a file containing all the services configurations & specifications.
+// As a quick workaround, we just need the service & url to get things working for now.
+func hackStringToServices(servicesArg string) ([]*sharedtypes.SupplierServiceConfig, error) {
+	supplierServiceConfig := make([]*sharedtypes.SupplierServiceConfig, 0)
+	serviceStrings := strings.Split(servicesArg, ",")
+	fmt.Println("OLSH", serviceStrings, servicesArg)
```

Comment:
Is this a debug line?

Reply:
Nah man, gotta leave a trail with your mark in the code ;)